.github/workflows/publish-docs.yml

```
@@ -90,18 +81,24 @@
               echo "ENVIRONMENT=preview" >> $GITHUB_ENV
               echo "DEPLOY_PROD=false" >> $GITHUB_ENV
           fi
-      - name: Pull Vercel Environment Information
-        run: vercel pull --yes --environment=${{ env.ENVIRONMENT }} --token=${{ secrets.VERCEL_TOKEN }}
+      - name: Remove .vercel directory
+        working-directory: docs/site
+        run: rm -rf .vercel
+      - name: Link Vercel Project
+        working-directory: docs/site
+        env:
+            VERCEL_PROJECT_ID: ${{ secrets.VERCEL_DOC_PROJECT_ID}}
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 10 but found 12

(indentation)